dev/benchmarks/microbenchmarks/lib/gestures/gesture_detector_bench.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '../common.dart';
+import './apps/button_matrix_app.dart' as button_matrix;
+
+const int _kNumWarmUpIters = 20;
+const int _kNumIters = 300;
+
+Future<void> main() async {
+  assert(false, "Don't run benchmarks in checked mode! Use 'flutter run --release'.");
```

Comment:
shouldn't this be `flutter run --profile`?

Reply:
This is also redundant with the check in `benchmarkWidgets`, or no?

Reply:
I'm using the same boilerplate with other benchmarks, such as https://github.com/flutter/flutter/blob/master/dev/benchmarks/microbenchmarks/lib/stocks/animation_bench.dart

Reply:
Ahh ok